src/appindexing.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+goog.provide('appindexing');
+
+goog.require('goog.json');// jshint unused:false
+goog.require('utils');
+
+
+appindexing.state = {};
+appindexing.state['androidAppIndexingTagsPresent'] = false;
+appindexing.state['iOSAppIndexingTagsPresent'] = false;
+appindexing.state['androidDetailsComplete'] = false;
+appindexing.state['iOSDetailsComplete'] = false;
+appindexing.state['addediOSTagDom'] = false;
+appindexing.state['addedAndroidTagDom'] = false;
+
+appindexing.checkForAppIndexingTags = function() {
```

Comment:
I might change this function name to `updateAppIndexingTagsIfPresent` which properly represents its purpose in life